packages/next/test/integration/integration-2.test.js

```diff
@@ -637,41 +635,12 @@ describe('rewrite headers', () => {
 
   it('should strip the hash from the rewritten path', () => {
     const route = routes.filter(r => r.src?.includes('suffix'));
-    expect(route.length).toBe(2);
+    expect(route.length).toBe(1);
     expect(route[0].headers).toEqual({
       'x-nextjs-rewritten-path': '/$1',
       'x-nextjs-rewritten-query': 'suffix=$1',
     });
   });
-
-  it('should not add rewrite headers when it is excluded with missing', () => {
-    const route = routes.filter(r => r.src?.includes('missing'));
-    expect(route.length).toBe(1);
-    expect(route[0].headers).toBeUndefined();
-  });
```

Comment:
Not sure what this test was intending to assert but are we certain this assertion is fine to drop?